components/Habit.js

```diff
@@ -36,26 +47,31 @@ export default function Habit(props) {
     <>
       { props.isEditable ?
         (
-          <div className='card' style={{ backgroundColor: props.color}} onClick={styleCheckmark}>
+          <div className='card' style={{ backgroundColor: props.color}} onClick={selectCard}>
             <div className="topLayer">
-              <div className='emojiContainer'>
-                  { emoji ? 
-                    (
-                      <>
-                        <div className='emoji' onClick={handleEmojiClick}>{ emoji }</div>
-                        <div className='shadow'></div>
-                      </>
-                    )
-                    :
-                    (
-                      <div className='addEmoji'>
-                        {
-                          emoji === "" ? <p className="addEmojiText">☻ Add icon</p> : <div className="emoji">{ emoji }</div>
-                        }
-                        <div className='shadow'></div>
-                      </div>
-                    )
-                  }
+              <div className='topLeft'>
+                <div className='redX'>
+                  <img src={'/redX.svg'} alt="SVG as an image" onClick={deleteHandler}/>
```

Comment:
both the delete SVG and the current placement don't look good. Can you point me to the original Dribble you were trying to follow? We need to find something that matches more of the theme you have.

Reply:
https://dribbble.com/shots/18932568-Morin-Habit-Tracker-App
Here's my dribble ^

In the dribble I followed, there was no button to delete habits.